indico/modules/events/registration/lists.py

```diff
@@ -54,11 +57,24 @@ def __init__(self, regform):
             'payment_date': {
                 'title': _('Payment date'),
             },
+            'tags_present': {
+                'title': _('Tags'),
+                'filter_choices': registration_tag_choices
+            },
+            'tags_absent': {
+                'title': _('Tags absent'),
```

Comment:
```suggestion
                'title': _('Without tags'),
```

I wonder if that's clearer... not 100% sure though

Reply:
'Absent tags', 'Missing tags', .. dunno :confused: